cl/search/models.py

```diff
@@ -1736,6 +1760,47 @@ def as_search_dict(self, docket_metadata=None):
 
         return normalize_search_dicts(out)
 
+    def get_csv_columns(self, get_column_name=False):
+        columns = [
+            "id",
+            "document_type",
+            "description",
+            "acms_document_guid",
+            "date_upload",
+            "document_number",
+            "attachment_number",
+            "pacer_doc_id",
+            "is_free_on_pacer",
+            "is_available",
+            "is_sealed",
+            "sha1",
+            "page_count",
+            "file_size",
+            "filepath_local",
+            "filepath_ia",
+            "ocr_status",
+        ]
+        if get_column_name:
+            columns = [self.add_class_name(col) for col in columns]
+        return columns
+
+    def _get_readable_document_type(self, *args, **kwargs):
+        return self.get_document_type_display()
+
+    def _get_readable_ocr_status(self, *args, **kwargs):
+        return self.get_ocr_status_display()
```

Comment:
Per Eduardo's comment, let's do one more of these for filepath_local. Notionally:

```
    def _get_full_filepath_local(self, *args, **kwargs):
        if self.filepath_local:
            return f"https://storage.courtlistener.com/{self.filepath_local}"
        return ""
```

Reply:
done

Reply:
[Uploading united-states-v-cadden.mad.4275782.2024-08-25.csv…]()
Explample with change in filepath if any